draft-ietf-quic-http.md

```diff
@@ -1102,18 +1109,33 @@ that can be prioritized are:
 
 Taken together, the dependencies across all prioritized elements in a connection
 form a dependency tree. An element can depend on another element or on the root
-of the tree. A reference to an element which is no longer in the tree is treated
-as a reference to the root of the tree. The structure of the dependency tree
-changes as PRIORITY frames modify the dependency links between prioritized
-elements.
-
-Due to reordering between streams, an element can also be prioritized which is
-not yet in the tree. Such elements are added to the tree with the requested
-priority.
+of the tree.  The tree also contains a dedicated placeholder with index "-1"
```

Comment:
Could we call it something else than "-1" placeholder? Maybe "orphan placeholder".

The reason I feel uncomfortable with calling it "-1" are:
* It is not actually an index.
* We already have a placeholder that is called by name "root".

Reply:
I prefer Kazuho's idea here about "-1".  Though I disagree about weight 0.  HTTP/2 didn't have it, which makes this more complicated.  I don't even think that you need weight 0, that's a gratuitous change in this PR.  You can simply say that streams with an "orphaned" priority is not given resources until all prioritized streams are handled.

Reply:
Clarification: my intent was to state what exactly @martinthomson recommends.